aws-nuke-config.yaml

```diff
@@ -0,0 +1,15 @@
+---
+regions:
+  - "us-west-2"
+
+account-blacklist:
+  - "we-dont-have-a-prod-account-to-blacklist-but-this-field-is-mandatory"
+
+accounts:
+  AWS_ACCOUNT_ID_REPLACE_ME: {}
+
+
+resource-types:
+  targets:
+  - CloudFormationStack
+  - EKSCluster
```

Comment:
Cloudformation for all eksctl resources
EKSCluster for non-eksctl created integration tests.

I will play around and see if EC2 has to be added, but it shouldn't need to be.